input/common/hid/device_wiiu_gca.c

```diff
@@ -116,15 +116,14 @@ static void update_pad_state(wiiu_gca_instance_t *instance)
 
    joypad_connection_t *pad;
    /* process each pad */
-   //RARCH_LOG_BUFFER(instance->device_state, 37);
    for(i = 1; i < 37; i += 9)
    {
       port = i / 9;
       pad = instance->pads[port];
 
-      port_connected = instance->device_state[i] & GCA_PORT_CONNECTED;
+      port_connected = instance->device_state[i];
 
-      if(port_connected)
+      if(port_connected > GCA_PORT_POWERED)
```

Comment:
`>=` ?

Reply:
Nope. `>` is correct.

`GCA_PORT_POWERED` (0x04) is a bit that is set if the GCA has both USB cables connected. It lets the driver know that rumble is available (although we don't presently use that functionality; I don't know if the rumble command has been figured out yet). So, that means that both `0x00` and `0x04` values mean "port is empty." Therefore, anything greater than 0x04 means something is plugged in.

This is purposefully generic because, at least for the time being, we don't actually need to distinguish the things plugged into the GCA.

Reply:
ok